src/main/java/Board.java

```diff
@@ -0,0 +1,117 @@
+import java.util.Arrays;
+
+public class Board {
+
+    private Player[][] grid;
+    private int gridSize = 3;
+
+    public Board() {
+        grid = new Player[gridSize][gridSize];
+        for (Player[] row : grid) {
+            Arrays.fill(row, Player.NEITHER);
+        }
+    }
+
+    public Player getSquareValue(int row, int col) {
+        return grid[row][col];
+    }
+
+    public void setSquareValue(int row, int col, Player player) {
```

Comment:
Perhaps there's a better word than Value that we could use here. Same for the method above.